src/components/WatchlistHistory/GetWatchlistHistory.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { graphql } from 'react-apollo'
+import { connect } from 'react-redux'
+import { TOTAL_TYPES } from './constants'
+import {
+  CATEGORY_HISTORY_QUERY,
+  PROJECTS_HISTORY_QUERY,
+  WATCHLIST_HISTORY_QUERY
+} from './WatchlistHistoryGQL'
+import WatchlistHistoryWidget from './WatchlistHistoryWidget'
+import { filterEmptyStats } from './utils'
+
+const getMarketcapQuery = ({ type, projects, range, byFunction }) => {
```

Comment:
byFunction used for true/false.
Is it ok name for this? @haritonasty

Reply:
isByFunction better?)